### ISSUE TITLE
Incorrect warning message for simulate function

### ISSUE THREAD

**Issue by smjenness** [statnet/ergm#18]
Running the following:

```r
library(ergm)
nw <- network.initialize(100, directed = FALSE)
fit <- ergm(nw ~ edges, target.stats = 50)
sim <- simulate(fit)
```

Now generates this warning message in the new `ergm`:
```r
Warning message:
You appear to be calling simulate.formula() directly. simulate.formula() is a method, and will not be exported in a future version of ‘ergm’. Use simulate() instead, or getS3method() if absolutely necessary.
```

This only shows up the first time the `simulate` function is run in an R session.

**Comment by krivit** [comment]
I could have sworn I fixed all of those... Now fixed in `master` and `3.9` branches.

**Comment by sgoodreau** [comment]
@krivit your comment from Aug 31 says this issue should be fixed in all `3.9` branches.  Alas I have `ergm 3.9.4` (current CRAN version) installed, and I still get it.

Code where it appears for me, along with session info:
```
 library(ergm)
 data("faux.mesa.high")
 fit <- ergm(faux.mesa.high~edges+nodematch("Race"))
 simnet <- simulate(fit, nsims=1)
```

```
> sessionInfo()
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252  LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252 LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] ergm_3.9.4       network_1.13.0.1

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.0           rstudioapi_0.8       bindr_0.1.1          magrittr_1.5        
 [5] MASS_7.3-50          tidyselect_0.2.5     lattice_0.20-35      R6_2.3.0            
 [9] rlang_0.3.0.1        dplyr_0.7.8          tools_3.5.1          parallel_3.5.1      
[13] grid_3.5.1           lpSolve_5.6.13       coda_0.19-2          yaml_2.2.0          
[17] assertthat_0.2.0     tibble_1.4.2         crayon_1.3.4         Matrix_1.2-14       
[21] bindrcpp_0.2.2       purrr_0.2.5          trust_0.1-7          glue_1.3.0          
[25] robustbase_0.93-3    statnet.common_4.1.4 DEoptimR_1.0-8       compiler_3.5.1      
[29] pillar_1.3.0         pkgconfig_2.0.2  
```

**Comment by krivit** [comment]
Branches, not releases. If you pull the 3.9 branch from the repository, it'll be fixed. However, there hasn't been a release from the 3.9 branch since 3.9.4.
